org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/properties/SonarLintExtraArgumentsPreferenceAndPropertyPage.java

```diff
@@ -458,7 +458,8 @@ protected void performDefaults() {
   }
 
   private IProject getProject() {
-    return getElement().getAdapter(IProject.class);
+    // note: the cast to IResource is necessary for e43 and e44
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unnecessary cast to "IProject". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1905)